lib/c-api/src/wasm_c_api/unstable/module.rs

```diff
@@ -150,3 +152,30 @@ pub unsafe extern "C" fn wasmer_module_set_name(
 
     module.inner.set_name(name)
 }
+
+/// A WebAssembly module contains stateless WebAssembly code that has
+/// already been compiled and can be instantiated multiple times.
+///
+/// Creates a new WebAssembly Module given the configuration
+/// in the engine.
```

Comment:
The comment doesn't look in sync with the code

Reply:
This was copied from `wasm_module_new`. Could you be more specific how exactly this is out of sync?

Reply:
You don't create a Module from a configuration here, you create a module from an Engine. That's what I meant.

Reply:
I am not sure that is how it is meant to be interpreted. To me this means the engine configuration influences the the module creation.

See other documentation like this: https://github.com/wasmerio/wasmer/blob/bb79130ccffaec08061f7574e9955e825f7c5ec8/lib/api/src/module.rs#L48-L49

I suppose one can make this more explicit. How about:

> Creates a new WebAssembly Module using the provided engine, respecting its configuration.

Reply:
I've changed this part of the docs.